traffic_monitor/manager/opsconfig.go

```diff
@@ -169,42 +164,26 @@ func StartOpsConfigManager(
 			backoff = util.NewConstantBackoff(util.ConstantBackoffDuration)
 		}
 		for {
-			realToSession, toAddr, err = to.LoginWithAgent(newOpsConfig.Url, newOpsConfig.Username, newOpsConfig.Password, newOpsConfig.Insecure, staticAppData.UserAgent, useCache, trafficOpsRequestTimeout)
+			err = toSession.Update(newOpsConfig.Url, newOpsConfig.Username, newOpsConfig.Password, newOpsConfig.Insecure, staticAppData.UserAgent, useCache, trafficOpsRequestTimeout)
 			if err != nil {
 				handleErr(fmt.Errorf("MonitorConfigPoller: error instantiating Session with traffic_ops (%v): %s\n", toAddr, err))
 				duration := backoff.BackoffDuration()
 				log.Errorf("retrying in %v\n", duration)
 				time.Sleep(duration)
 
 				if toSession.BackupFileExists() && (toLoginCount >= cfg.TrafficOpsDiskRetryMax) {
-					jar, err := cookiejar.New(nil)
-					if err != nil {
-						log.Errorf("Err getting cookiejar")
-						continue
-					}
-
-					realToSession = to.NewSession(newOpsConfig.Username, newOpsConfig.Password, newOpsConfig.Url, staticAppData.UserAgent, &http.Client{
-						Timeout: trafficOpsRequestTimeout,
-						Transport: &http.Transport{
-							TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
-						},
-						Jar: jar,
-					}, useCache)
-					toSession.Set(realToSession)
-					// At this point we have a valid 'dummy' session. This will allow us to pull from disk but will also retry when TO comes up
```

Comment:
Are you sure removing this won't break TM when TO is unaccessible but the backup file exists?

Reply:
No, but tbh TM already seems broken in that case, from my testing with your UI indicator. Have I been doing something wrong? Does it work for you?

Reply:
Depends on what you mean by working. In a strict sense, no it is not. 

After further review you should be good on this, it looks like it was originally just a copy paste of the Login function anyways.

This shouldn't be an issue, but I think it's worth noting: With these changes (assuming TO is down), each attempt will take twice as long since it tries both legacy and new TO.

Reply:
Yeah, that's unavoidable, since we don't have a way to request which API versions TO supports. There was a proposal for that, but it never really got off the ground.

Reply:
Issue about TM not working with backup files captured in #4946 .